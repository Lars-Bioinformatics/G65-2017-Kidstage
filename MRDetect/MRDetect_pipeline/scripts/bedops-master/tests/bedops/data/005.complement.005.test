chr1	0	1
chr1	4	5
chr10	0	1
chr10	20	30
