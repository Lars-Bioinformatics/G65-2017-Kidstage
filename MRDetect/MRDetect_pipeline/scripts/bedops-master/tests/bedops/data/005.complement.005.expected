chr1	1	4
chr10	1	20
